package.json

```diff
@@ -17,35 +17,22 @@
     "analytics",
     "analytics-manager"
   ],
-  "main": "commonjs/index.js",
-  "module": "module/index.js",
```

Comment:
@robgonnella deleting module property won't support module and tools like @pikapkg or bundlephobia work properly in your package

Reply:
Once again thanks for reporting! Just added module support back and published 2.1.5
https://github.com/robgonnella/redux-analytics-manager/commit/ebc014185fae130ad49f312a1801a5c7e63b0b7b